pkgs/games/freeorion/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchgit, cmake, boost, SDL2, python2, freetype, openal, libogg, libvorbis, zlib, libpng, libtiff, libjpeg, mesa, glew, git, doxygen }:
+
+stdenv.mkDerivation rec {
+  version = "2016-04-16";
+  name = "freeorion-${version}";
+
+  src = fetchgit {
+    url = "https://github.com/freeorion/freeorion";
```

Comment:
It's certainly possible something is packaged the wrong way. Try using `strace -f` and look for errors, particularly `ENOENT` -- this should help you if the problem is nonexistent file. Also looking through the game logs should be much easier, but that is only if the game is actually writing some kind of error there.